pkg/endpoint/bpf.go

```diff
@@ -63,19 +64,21 @@ func (e *Endpoint) writeL4Map(fw *bufio.Writer, owner Owner, m policy.L4PolicyMa
 		}
 
 		redirect = common.Swab16(redirect)
-		entry := fmt.Sprintf("{%d,%d,%d}", dport, redirect, protoNum)
+		entry := fmt.Sprintf("%d,%d,%d,%d", index, dport, redirect, protoNum)
 		if array != "" {
 			array = array + "," + entry
 		} else {
 			array = entry
 		}
 
+		index += 1
```

Comment:
@jrfastab If the intention is to use the index of the array you can replace the line 49 from `for _, l4 := range m {` to `for i, l4 := range m {`